homework24/src/ba/bitcamp/homework24/task2/CustomMessages/Message.java

```diff
@@ -0,0 +1,60 @@
+package ba.bitcamp.homework24.task2.CustomMessages;
+
+import javax.swing.JOptionPane;
+
+public class Message {
+
+	/**
+	 * Public method that returns user selection if message is warning type,
+	 * otherwise returns -50 number. Opens different JOptionPane depending on
+	 * inputed type as well as inputed message.
+	 * 
+	 * @param type
+	 *            <code>String</code> type value of type of message, error or
+	 *            warning
+	 * @param message
+	 *            <code>String</code> type value of message
+	 * @return <code>int</code> type value of user choice if no choice is made
+	 *         -50 is returned
+	 */
+	public static int message(String type, String message) {
+		if (type.equals("error")) {
```

Comment:
Znaci ako si vec pravio konstante zasto ih nisi izvukao u neku klasu i rekao sada npr, ErrorMessages.ERROR.equals(type). Druga notifikacija je da uvijek stavis na prvo mjesto nesta sta je sigurno da postoji npr, siguran si uvijek da postoji konstanta dok to nisi siguran sa type varijablom. Zamisli samo sta bi se desilo ovom kodu da iz nekog razloga dodje type da je null bilo bi null.equals("error") i dobio bi NullPointException ;).